modules/rosa-edit-objects.adoc

```diff
@@ -0,0 +1,194 @@
+:_module-type: REFERENCE
+// Module included in the following assemblies:
+//
+// * cli_reference/rosa_cli/rosa-manage-objects-cli.adoc
+
+[id="rosa-edit-objects_{context}"]
+= Edit objects
+
+[role="_abstract"]
+This section describes the `edit` commands for clusters and resources.
+
+[id="rosa-edit-cluster_{context}"]
+== edit cluster
+
+Allows edits to an existing cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa edit cluster [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--cluster
+|(Required) The name or ID (string) of the cluster to edit.
+
+|--private
+|Restricts a master API endpoint to direct, private connectivity.
+
+|--enable-cluster-admins
```

Comment:
Should this be `cluster-admin` singular?

Reply:
That is removed now per QE.